src/core/ezcurl_core.ml

```diff
@@ -217,6 +229,20 @@ let string_of_meth = function
 
 let pp_meth out m = Format.pp_print_string out (string_of_meth m)
 
+type sse_frame = {
+  event: string option;
+  id: string option;
+  data: string option;
+  retry: int option;
+  empties: string list; (* Lines without a ':' *)
```

Comment:
still needs to be removed! :)

(afaik the browser API removes these, so it's unlikely anyone actually relies on it)